packages/accordion/src/AccordionContainer.spec.tsx

```diff
@@ -9,43 +9,43 @@ import React from 'react';
 import { render, fireEvent } from '@testing-library/react';
 import { KEY_CODES } from '@zendeskgarden/container-utilities';
 import { AccordionContainer } from './AccordionContainer';
+import { IUseAccordionReturnValue } from './useAccordion';
 
 describe('AccordionContainer', () => {
   it('renders with expected return values', () => {
-    const { getByTestId } = render(
+    const { getByTestId, getByText } = render(
       <AccordionContainer>
-        {({ expandedSections, disabledSections }) => (
-          <div
-            data-test-id="test"
-            hidden={expandedSections.length === 0}
-            disabled={disabledSections.length !== 0}
```

Comment:
TypeScript was not allowing `disabled` to be applied onto a `div`. 

I added a `<button>` to this test and applied the `disabled` attribute to it. This also better reflects how `<AccordionContainer>` is actually used — the `disabled` attribute is applied to the button / trigger.

Reply:
I think TypeScript has this one wrong. https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/hidden.

Reply:
:eyes: Could you clarify what you mean by TypeScript might have this one wrong? I found the type [definition](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L1646) on DefinitelyTyped and it seems to match with the use case (`hidden` being a boolean). Anything I missed or should be aware of?

Reply:
I read your comment wrong – I was thinking `hidden` but you are clearly talking about `disabled`. Carry on :wink: